library/testing/CMakeLists.txt

```diff
@@ -82,6 +82,8 @@ set_target_properties(libf3dSDKTests PROPERTIES
   CXX_VISIBILITY_PRESET hidden
   )
 
+target_compile_definitions(libf3dSDKTests PRIVATE F3D_MODULE_EXR=$<BOOL:${F3D_MODULE_EXR}>)
```

Comment:
so do you need to remove it from the config file then ?

Reply:
I'll do it in another PR

Reply:
could you use config in this one and cleanup everything in another ? I'd rather not have a partial use of this.

Reply:
No sorry it's not possible, I don't have access to the config in the test. That's one of the reason I had to change the method.

Reply:
I'd rather do the migration everywhere if you insist, but I think this PR is already big enough

Reply:
lets not do it but add a TODO to not forget about it